core/src/main/res/layout/tag_content.xml

```diff
@@ -8,26 +8,31 @@
     style="@style/Widget.KiwixTheme.Chip.Choice.Static"
     android:layout_width="wrap_content"
     android:layout_height="wrap_content"
-    android:text="@string/tag_pic" />
+    android:text="@string/tag_pic"
+    android:textAlignment="center" />
+
 
   <com.google.android.material.chip.Chip
     android:id="@+id/tag_video"
     style="@style/Widget.KiwixTheme.Chip.Choice.Static"
     android:layout_width="wrap_content"
     android:layout_height="wrap_content"
-    android:text="@string/tag_vid" />
+    android:text="@string/tag_vid"
+    android:textAlignment="center" />
 
   <com.google.android.material.chip.Chip
     android:id="@+id/tag_short_text"
     style="@style/Widget.KiwixTheme.Chip.Choice.Static"
     android:layout_width="wrap_content"
     android:layout_height="wrap_content"
-    android:text="@string/tag_short_text" />
+    android:text="@string/tag_short_text"
+    android:textAlignment="center" />
 
   <com.google.android.material.chip.Chip
     android:id="@+id/tag_text_only"
     style="@style/Widget.KiwixTheme.Chip.Choice.Static"
     android:layout_width="wrap_content"
     android:layout_height="wrap_content"
-    android:text="@string/tag_text_only" />
+    android:text="@string/tag_text_only"
+    android:textAlignment="center" />
```

Comment:
You can move this into our style `Widget.KiwixTheme.Chip.Choice.Static`, just ctrl+click on it to go to the declaration

Reply:
Got it!

Reply:
Hey @macgills, while attempting moving it to `Widget.KiwixTheme.Chip.Choice.Static`, I got an error that `android:textAlignment` requires minSdk 17 or above (we're at 15), and also found out the text isn't centered align for only Pic and Vid as their number of letters is <= 3. Would it be alright to leave the change at this folder but for Pic and Vid components only?

Reply:
press `alt+enter` on the red warning and it will give you the option to add a corresponding attribute for lower versions, `gravity= "center_horizontal"` sounds right to me. Put both in the style, I want this style to handle displaying chips for me in any scenario

Reply:
Tried that as well hence I suggested the above, 
`<item name="android:gravity">center_horizontal</item>` doesn't have the intended effect as `wrap_content` is used in the chip layout

Reply:
```
    <item name="android:textAlignment" tools:ignore="NewApi">center</item>
```

go for that, if we can't do the proper thing for api <17 I don't think it matters, the issue is minor and who knows for how much longer we will bother supporting below 17. I'd prefer if we set min to 19(at least) but that is a discussion for another time

Reply:
Cool didn't know about that trick, thank you! And yea definitely, that decision would need a discussion.